packages/editor-ui/src/types/router.ts

```diff
@@ -46,6 +46,7 @@ export interface RouteConfig {
 		};
 		scrollOffset?: number;
 		setScrollPosition?: (position: number) => void;
+		readonly?: boolean;
```

Comment:
Suggestion - maybe `readOnlyCanvas` to be specific? Since this should only affect the canvas.

Reply:
Makes sense, thanks!